### PR TITLE
Extend csv config

### PR DESCRIPTION
**Pull request by vtretyakov**
* Added units to the comments
* Added missing feedback sensors

### CODE REVIEW

File: examples/app_demo_master_cyclic/sdo_config/sdo_config.csv [hunk 1]
```diff
@@ -1,87 +1,144 @@
...
+0x2202,       10,           8,           8,           8,           8,           8,           8 # BISS_ENCODER_CLOCK_PORT_CONFIG - which HW port is used for clock
+0x2202,       12,           0,           0,           0,           0,           0,           0 # BISS_ENCODER_NUMBER_OF_FILLING_BITS [bits]
+0x2202,       13,          30,          30,          30,          30,          30,          30 # BISS_ENCODER_NUMBER_OF_BITS_TO_READ_WHILE_BUSY [bits]
+#REM 16MT Encoder
+0x2203,        1,           6,           6,           6,           6,           6,           6 # SENSOR TYPE (6 is REM 16MT)
+0x2203,        2,           1,           1,           1,           1,           1,           1 # ENCODER_FUNCTION - 0: Disabled, 1: Commutation and motion control, 2: Commutation and feedback display only, 3: Motion Control, 4: Feedback display only, 5: Commutation only
+0x2203,        3,      262144,      262144,      262144,      262144,      262144,      262144 # ENCODER_RESOLUTION [encoder increments (ticks)]
```
Comment (romu4ld) [inline examples/app_demo_master_cyclic/sdo_config/sdo_config.csv]: REM 16MT resolution is 16bit so 65536

File: examples/app_demo_master_cyclic/sdo_config/sdo_config.csv [hunk 2]
```diff
@@ -1,87 +1,144 @@
...
+0x2201,        3,      262144,      262144,      262144,      262144,      262144,      262144 # ENCODER_RESOLUTION [encoder increments (ticks)]
+0x2201,        4,          50,          50,          50,          50,          50,          50 # ENCODER_VELOCITY_CALCULATION_PERIOD [usec]
+0x2201,        5,          0,            0,           0,           0,           0,           0 # ENCODER_POLARITY (0 normal,  1 inverted)
+0x2201,        6,          10,          10,          10,          10,          10,          10 # BISS_ENCODER_MULTITURN_RESOLUTION [bits]
+0x2201,        7,        4000,        4000,        4000,        4000,        4000,        4000 # BISS_ENCODER_CLOCK_FREQUENCY [Hz]
+0x2201,        8,          20,          20,          20,          20,          20,          20 # BISS_ENCODER_TIMEOUT - BiSS timeout in [usec]
+0x2201,        9,        0x30,        0x30,        0x30,        0x30,        0x30,        0x30 # BISS_ENCODER_CRC_POLYNOM - polynom in reverse representation:  x^0 + x^1 + x^6 is 0b110000
+0x2201,       10,           8,           8,           8,           8,           8,           8 # BISS_ENCODER_CLOCK_PORT_CONFIG - which HW port is used for clock
```
Comment (romu4ld) [inline examples/app_demo_master_cyclic/sdo_config/sdo_config.csv]: The subindex 11 is missing (SUB_BISS_ENCODER_DATA_PORT_CONFIG)
It was not needed in master but it is develop and I forgot to add it.

Comment (vtretyakov) [inline examples/app_demo_master_cyclic/sdo_config/sdo_config.csv]: What are the options for configuring this port?

Comment (romu4ld) [inline examples/app_demo_master_cyclic/sdo_config/sdo_config.csv]: ENCODER_PORT_1 = 0,         /**< Encoder port 1 */
    ENCODER_PORT_2 = 1,         /**< Encoder port 0 */
    ENCODER_PORT_EXT_D0 = 2,    /**< Encoder port GPIO O */
    ENCODER_PORT_EXT_D1 = 3,    /**< Encoder port GPIO 1 */
    ENCODER_PORT_EXT_D2 = 4,    /**< Encoder port GPIO 2 */
    ENCODER_PORT_EXT_D3 = 5     /**< Encoder port GPIO 3 */

Comment (romu4ld) [inline examples/app_demo_master_cyclic/sdo_config/sdo_config.csv]: We can add the possible values for clock port config:
    BISS_CLOCK_PORT_EXT_D0=0,       /**< GPIO port number 0 */
    BISS_CLOCK_PORT_EXT_D1=1,       /**< GPIO port number 1 */
    BISS_CLOCK_PORT_EXT_D2=2,       /**< GPIO port number 2 */
    BISS_CLOCK_PORT_EXT_D3=3,       /**< GPIO port number 3 */
    BISS_CLOCK_PORT_EXT_D4=4,  /**< hall_enc_select_port 3rd bit */
    BISS_CLOCK_PORT_EXT_D5=8   /**< hall_enc_select_port 4th bit */
and 8 is default

File: examples/app_demo_master_cyclic/sdo_config/sdo_config.csv [hunk 3]
```diff
@@ -108,15 +110,15 @@
 0x2205,        3,        4000,        4000,        4000,        4000,        4000,      262144 # ENCODER_RESOLUTION [encoder increments (ticks)]
 0x2205,        4,        1000,        1000,        1000,        1000,        1000,        1000 # ENCODER_VELOCITY_CALCULATION_PERIOD [usec]
 0x2205,        5,           0,           0,           0,           0,           0,           0 # ENCODER_POLARITY (0 normal,  1 inverted)
-0x2205,        6,           3,           3,           3,           3,           3,           3 # ENCODER_NUMBER_OF_CHANNELS [2:3] - 2: AB, 3: ABI
+0x2205,        6,           3,           3,           3,           3,           3,           3 # ENCODER_NUMBER_OF_CHANNELS [0:1] - 0: AB (2 channels), 1: ABI (3 channels)
```
Comment (romu4ld) [inline examples/app_demo_master_cyclic/sdo_config/sdo_config.csv]: You didn't update the values in the columns. The default should be 1 (qei with index)

Comment (vtretyakov) [inline examples/app_demo_master_cyclic/sdo_config/sdo_config.csv]: Oh, thanks!

**Pull request merged**